shared/constants/profile.js

```diff
@@ -20,22 +21,42 @@ export type WaitingRevokeProof = TypedAction<'profile:revoke:waiting', {waiting:
 export const finishRevokeProof = 'profile:revoke:finish'
 export type FinishRevokeProof = TypedAction<'profile:revoke:finish', void, {error: string}>
 
+export const updateProofText = 'profile:updateProofText'
+export type UpdateProofText = TypedAction<'profile:updateProofText', {proof: string}, void>
+
+export const updateError = 'profile:updateError'
+export type UpdateError = TypedAction<'profile:updateError', {error: ?string, errorCode: number}, void>
```

Comment:
well this pattern is a little different. there are a bunch of actions which can emit this but its not necessarily an error in the action. i guess we should discuss